libexec/goenv

```diff
@@ -118,6 +118,13 @@ case "$command" in
   exec goenv-help
   ;;
 * )
+  if [ "$command" = "shell" ] && [ -z "${GOENV_SHELL}" ]; then
+    echo 'eval "$(goenv init -)" has not been executed.'
+    echo "Please read the installation instructions in the README.md at github.com/syndbg/goenv"
+    echo "or run 'goenv help init' for more information"
+    exit 1
+  fi
+
   command_path="$(command -v "goenv-$command" || true)"
```

Comment:
Sorry for going back to this old issue. I'm a bit confused. Shouldn't here be `goenv-sh-$command`? I don't see a `goenv-shell` under libexec, but only goenv-sh-shell.

Reply:
Sorry never mind, I did not do `eval "$(goenv init -)"` properly so `goenv` did not end up with a shell function.